docs/ops/activation/Clamp_1.md

```diff
@@ -6,48 +6,66 @@
 
 **Short description**: *Clamp* operation represents clipping activation function.
 
+**Detailed description**:
+
+*Clamp* performs clipping operation over the input tensor element-wise. Element values of the output are within the range `[min, max]`.
+* Input values that are smaller than *min* are replaced with *min* value. For example, *min* equal 10 means that any value in the input that is smaller than the bound, is replaced by 10.
+* Input values that are greater than *max* are replaced with the *max* value. For example, *max* equals 50 means that any value in the input that is greater than the bound, is replaced by 50.
+* Input values within the range `[min, max]` remain unchanged.
+
+Mathematical formula of *Clamp* is as follows:
+\f[
+clamp( x )=\left\{\begin{array}{ll}
+    input( x ) \quad \mbox{if } \quad min \leq input( x ) \leq max \\
+    max \quad \mbox{if } \quad input( x ) > max \\
+    min \quad \mbox{if } \quad input( x ) < min
+\end{array}\right.
+\f]
+
 **Attributes**:
 
 * *min*
 
-  * **Description**: *min* is the lower bound of values in the output. Any value in the input that is smaller than the bound, is replaced with the *min* value. For example, *min* equal 10 means that any value in the input that is smaller than the bound, is replaced by 10.
-  * **Range of values**: non-negative positive floating point number
-  * **Type**: float
+  * **Description**: *min* is the lower bound of values in the output.
+  * **Range of values**: positive floating point number
```

Comment:
There is a list of [tests](https://github.com/openvinotoolkit/openvino/blob/master/ngraph/test/backend/fused_op.in.cpp#L483) which have negative min/max attributes, contradicting this statement:
- fused_clamp_double
- fused_clamp_float
- fused_clamp_int8
- fused_clamp_int16
- fused_clamp_int32
- fused_clamp_int64
- fused_clamp_uint8
- fused_clamp_uint16
- fused_clamp_uint32
- fused_clamp_uint64
- fused_clamp_float16
- fused_clamp_bfloat16

Other frameworks such as [pytorch](https://pytorch.org/docs/stable/generated/torch.clamp.html) and [onnx runtime](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/test/providers/cpu/math/clip_test.cc) allow negative values for attributes.
@vgavrilo Do you have any comments here?

Reply:
After discussion we concluded that attributes may be negative.